docs/20newsgroup.md

```diff
@@ -0,0 +1,66 @@
+# Anserini: 20-Newsgroup
```

Comment:
rename `experiments-20newsgroups.md` for consistency with everything else in that directory?

Reply:
"20 Newsgroups", without dash?

Reply:
So `experiments-20newsgroups.md`?